packages/aws-arch/scripts/common.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+export const GENERATED_DIR = path.resolve(__dirname, '..', 'src', 'generated');
+
+// Ensure the generated dir exists
+fs.ensureDirSync(GENERATED_DIR);
+
+export function logCount(title: string, total: number | object | any[] | Set<any>, count: number | object | any[] | Set<any>, delta: boolean = false): void {
```

Comment:
can we make the type for total a type variable and re-use it here and on line 18?